core/input/input.cpp

```diff
@@ -298,13 +298,13 @@ bool Input::is_action_just_pressed(const StringName &p_action, bool p_exact) con
 	}
 
 	if (Engine::get_singleton()->is_in_physics_frame()) {
-		return E->value.pressed && E->value.physics_frame == Engine::get_singleton()->get_physics_frames();
+		return p_current ? E->value.pressed : true && E->value.pressed_physics_frame == Engine::get_singleton()->get_physics_frames();
```

Comment:
Those `true &&` look like left-behinds of code iteration.

Reply:
Ah yes this must have been a mistake when refactoring (blame my old eyesight! :smile: )

I'll change this, but on relooking after a month gap, I'm now tending towards breaking compat for this bug and removing the `p_current` argument. I'm struggling to think of when users might want to have `is_action_just_pressed()` not trigger when they have just pressed the action, and it is possible to mimic the old case anyway. Will change.